hadoop-ozone/dev-support/checks/build.sh

```diff
@@ -18,7 +18,7 @@ cd "$DIR/../../.." || exit 1
 
 : ${OZONE_WITH_COVERAGE:="false"}
 
-MAVEN_OPTIONS='-V -B -Dmaven.javadoc.skip=true -DskipTests -DskipDocs --no-transfer-progress'
+MAVEN_OPTIONS='-V -B -DskipTests -DskipDocs --no-transfer-progress'
```

Comment:
build.sh is invoked in other places like intermittent-test-check.yml, repeat-acceptance.yml, build-ratis.yml . We would need to update skipjavadoc param in these places to retain same behaviour or undo this change

Reply:
Thanks for the reminder, updated.

Note that `build-ratis` invokes `build.sh` which comes with Ratis, not this one.